src/testdrive/src/action.rs

```diff
@@ -168,6 +168,7 @@ pub struct State {
     // === Testdrive state. ===
     arg_vars: BTreeMap<String, String>,
     cmd_vars: BTreeMap<String, String>,
+    default_arg_vars: BTreeMap<String, String>,
```

Comment:
I don't think you need a new map for this! I think we should be able to just set the value in `arg_vars` if it doesn't already exist.